xs/src/libslic3r/ConfigBase.hpp

```diff
@@ -706,18 +750,29 @@ class ConfigBase
     virtual ~ConfigBase() {};
```

Comment:
We should consider overriding Slic3r::ConfigBase&(const Slic3r::ConfigBase&) and operator=(const Slic3r::ConfigBase&) or deleting them as we have pointer data members, just to make sure the usage is obvious (and that the pointer is copied (as ConfigBase doesn't own its def). 
(-Weffc++)

Reply:
True.

Reply:
Deleting is no good; I've defined the relevant methods and set them to = default .